modules/retail/pages/retail-mass-config.adoc

```diff
@@ -3,6 +3,11 @@
 
 Mass configuration is possible with branch servers and terminals.
 
+[IMPORTANT]
+====
+Mass configuration tool was not yet adapted for containerized saltboot workflow.
```

Comment:
What does this mean?  Is it now adopted?  We do not warn about things not done optimal in the past `;)`

Reply:
This means that retail_yaml and retail_branch_init tools do not work with containerized branches. Only with non-containerized 4.3 branches.